src/main/java/org/spongepowered/api/command/args/GenericArguments.java

```diff
@@ -929,14 +929,14 @@ public static CommandElement bool(Text key) {
 
         @Override
         protected Iterable<String> getChoices(CommandSource source) {
-            return Arrays.asList(this.type.getEnumConstants()).stream()
+            return Arrays.stream(this.type.getEnumConstants())
                 .map(input -> input == null ? null : input.name())
```

Comment:
You can optimize this line to 
```java
.map(input::name)
```
since the elements of the array is guaranteed to be non-null.